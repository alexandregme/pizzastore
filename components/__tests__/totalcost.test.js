import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import TotalCost from '../totalcost';

describe("Total Cost", ()=>{
  let mountedTotalCost;
  beforeEach(()=>{
    mountedTotalCost = shallow(<TotalCost />);
  });

  it('renders without crashing', () => {
    shallow(<TotalCost />);
    expect(mountedTotalCost).toMatchSnapshot();
  });
});

