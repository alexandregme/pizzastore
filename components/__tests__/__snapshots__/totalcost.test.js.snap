// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Total Cost renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TotalCost />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "total cost - $1000.00",
      "className": undefined,
    },
    "ref": null,
    "rendered": "total cost - $1000.00",
    "type": "p",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "total cost - $1000.00",
        "className": undefined,
      },
      "ref": null,
      "rendered": "total cost - $1000.00",
      "type": "p",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
